blockchain/storagev2/leveldb/leveldb.go

```diff
@@ -44,21 +41,8 @@ func NewLevelDBStorage(path string, logger hclog.Logger) (*storagev2.Storage, er
 		return nil, err
 	}
 
-	// Open Lookup
-	// Set default options
-	options = &opt.Options{
-		BlockCacheCapacity: 64 * opt.MiB,
-		WriteBuffer:        opt.DefaultWriteBuffer,
-	}
-	path += "/lookup"
-
-	lookup, err := openLevelDBStorage(path, options)
-	if err != nil {
-		return nil, err
-	}
-
 	ldbs[0] = &levelDB{maindb}
-	ldbs[1] = &levelDB{lookup}
+	ldbs[1] = nil
```

Comment:
I need some indication do we have 2 instances or one. If we have only one, then we should always default to index 0 for all read/write operations. So if the 2nd element of array is nil then we know we should pickup db instance with index 0. With this dummy interface implementation we can't accomplish that

Method doing database mapping:
func (s *Storage) getDB(t uint8) Database {
	i := getIndex(t)
	if s.db[i] != nil {
		return s.db[i]
	}

	return s.db[MAINDB_INDEX]
}

Reply:
But again, instead of checking if it is nil, you can check whether the concrete type is the `NoOpDatabase`. But in the other places, it would be safe to invoke the function against the instance, because it would not be `nil`.

Reply:
Maybe, but that way we will not know if we are calling a bad instance or a good one. In my opinion it is better to crash then to call interface doing nothing. But anyway it will never happen to call a nil instance because all calls must go through this getDB mapper first.